components/no_results_indicator/no_results_indicator.scss

```diff
@@ -15,6 +15,7 @@
     text-align: center;
     word-break: break-word;
     max-width: 500px;
+    margin: auto;
```

Comment:
is `margin: auto` correct or should it rather be `margin: 0 auto`? Just asking out of curiosity!

Reply:
I think auto works fine as we will probably be dependent on the parent height in some scenarios